src/serviceconnector-passwordless/azext_serviceconnector_passwordless/_client_factory.py

```diff
@@ -2,22 +2,17 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
+import os
 
 
 # pylint: disable=consider-using-f-string
 def cf_connection_cl(cli_ctx, *_):
     from azure.mgmt.servicelinker import ServiceLinkerManagementClient
-    from azure.cli.core.commands.client_factory import get_mgmt_service_client, get_az_user_agent
-    from azure.core.pipeline import policies
+    from azure.cli.core.commands.client_factory import get_mgmt_service_client
     from .config import NAME, VERSION
-
-    user_agent_policy = policies.UserAgentPolicy(
-        user_agent=get_az_user_agent())
-    user_agent_policy.add_user_agent(
-        "CliExtension/{}({})".format(NAME, VERSION))
+    os.environ['AZURE_HTTP_USER_AGENT'] = "CliExtension/{}({})".format(NAME, VERSION)
```

Comment:
This would overwrite the existing value of `AZURE_HTTP_USER_AGENT`

Reply:
updated